docker-compose.yml

```diff
@@ -1,8 +1,10 @@
-version: '2'
+version: "2"
 
 services:
   postgres:
     image: postgres:9.4
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
This is just an example config, I'm not sure it's actively used anywhere in this repo directly.